### PR TITLE
Patient view improvements

### PR DESCRIPTION
**Pull request by evantyip**

### CODE REVIEW

**Comment by evantyip** [comment]
[Frontend: [feature] (patient view) separation of past & upcoming appointment cards](https://trello.com/c/zlCAj3S7/8-frontend-feature-patient-view-separation-of-past-upcoming-appointment-cards)

**Comment by evantyip** [comment]
[Frontend: [feature] (patient view) separation of past & upcoming appointment cards](https://trello.com/c/zlCAj3S7/8-frontend-feature-patient-view-separation-of-past-upcoming-appointment-cards)

File: pages/patient/index.tsx [hunk 1]
```diff
@@ -25,21 +26,55 @@ const PatientHome = () => {
...
+                    w-fit
+                    rounded-md
+                    border-gray-300
+                    shadow-sm
+                    focus:border-purple-300 focus:ring focus:ring-purple-200 focus:ring-opacity-50
+                  "
+          value={filter}
+          onChange={(e) => setFilter(e.target.value)}
+        >
+          <option>Upcoming</option>
```
Comment (htransteven) [inline pages/patient/index.tsx:200]: Do you need to set the `value` prop on the `<option>` element?

**Review by htransteven — APPROVED** [review]

File: pages/patient/index.tsx (hunk 1 above)
Comment (evantyip) [reply pages/patient/index.tsx:200]: You right I don't need to set it since I have default value for `useState`

**Pull request merged**